web/src/engine/osk/build.sh

```diff
@@ -30,13 +30,19 @@ builder_describe "Builds the Keyman Engine for Web's On-Screen Keyboard package
 
 builder_describe_outputs \
   configure   /node_modules \
+  configure   /web/src/resources/osk/keymanweb-osk.ttf \
```

Comment:
Wait... can the builder-script setup actually support two separate lines for the same `action:target` without losing one of them?  If so... wish I'd known that; there's a lot of Web scripts that could use multiple output-checks.

Reply:
I'm hoping that syntax works. builder.inc failed when I originally had
```
  configure  /node_modules \
             /web/src/resources/osk/keymanweb-osk.ttf \
  build ...
```

Reply:
No, currently it's only one output per line. Anything more is a future builder feature.